openmetadata-ui/src/main/resources/ui/src/utils/EntityUtils.tsx

```diff
@@ -2393,3 +2393,16 @@ export const getColumnSorter = <T, K extends keyof T>(field: K) => {
     return 0;
   };
 };
+
+export const highlightSearchText = (
+  text?: string,
+  searchText?: string
+): string => {
+  if (!searchText || !text) {
+    return text ?? '';
+  }
+
+  const regex = new RegExp(`(${searchText})`, 'gi');
+
+  return text.replace(regex, `<span class="text-highlighter">$1</span>`);
```

Comment:
Can we return HTML from here? Instead again using another function to do it?
highlightSearchText won't make sense to return string as we want highlight in HTML right?